### PR TITLE
Miscellaneous code inspection fixes

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: reporter/src/funTest/kotlin/reporters/EvaluatedModelReporterFunTest.kt [hunk 1]
```diff
@@ -67,7 +59,7 @@ private fun generateReport(reporter: EvaluatedModelReporter, ortResult: OrtResul
     val input = ReporterInput(
         ortResult = ortResult,
         resolutionProvider = DefaultResolutionProvider().add(ortResult.getResolutions()),
-        howToFixTextProvider = HOW_TO_FIX_TEXT_PROVIDER
+        howToFixTextProvider = { "Some how to fix text".trimIndent() }
```
Comment (sschuberth) [inline reporter/src/funTest/kotlin/reporters/EvaluatedModelReporterFunTest.kt:62]: Can't we remove the `.trimIndent()` here?

File: reporter/src/funTest/kotlin/reporters/StaticHtmlReporterFunTest.kt [hunk 2]
```diff
@@ -38,15 +38,13 @@ import org.ossreviewtoolkit.utils.ORT_NAME
...
-private val HOW_TO_FIX_TEXT_PROVIDER: HowToFixTextProvider = object : HowToFixTextProvider {
-    override fun getHowToFixText(issue: OrtIssue): String? {
-        return """
-            * *Step 1*
-            * __Step 2__
-            * ***Step 3***
-            ```Some long issue resolution text to verify that overflow:scroll is working as expected.``` 
-        """.trimIndent()
-    }
+private val HOW_TO_FIX_TEXT_PROVIDER: HowToFixTextProvider = HowToFixTextProvider {
```
Comment (sschuberth) [inline reporter/src/funTest/kotlin/reporters/StaticHtmlReporterFunTest.kt:41]: Maybe drop the explicit type here as `: HowToFixTextProvider = HowToFixTextProvider` reads a bit verbose?

File: reporter/src/main/kotlin/reporters/ExcelReporter.kt [hunk 3]
```diff
@@ -113,7 +113,7 @@ class ExcelReporter : Reporter {
         val workbook = XSSFWorkbook()
 
         defaultStyle = workbook.createCellStyle().apply {
-            setVerticalAlignment(VerticalAlignment.TOP)
+            verticalAlignment = VerticalAlignment.TOP
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/ExcelReporter.kt:116]: Actually, I believe the inspection is wrong here, as I recall this did not compile. Let's see what the CI checks say.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: reporter/src/funTest/kotlin/reporters/EvaluatedModelReporterFunTest.kt (hunk 1 above)
Comment (fviernau) [reply reporter/src/funTest/kotlin/reporters/EvaluatedModelReporterFunTest.kt:62]: Done.

File: reporter/src/funTest/kotlin/reporters/StaticHtmlReporterFunTest.kt (hunk 2 above)
Comment (fviernau) [reply reporter/src/funTest/kotlin/reporters/StaticHtmlReporterFunTest.kt:41]: Done.

**Review by sschuberth — DISMISSED** [review]

**Review by mnonnenmacher — DISMISSED** [review]

File: reporter/src/main/kotlin/reporters/ExcelReporter.kt (hunk 3 above)
Comment (mnonnenmacher) [reply reporter/src/main/kotlin/reporters/ExcelReporter.kt:116]: I remember the same, but it was a long time and several Kotlin versions ago, maybe something was improved in the meantime.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**